frontend/src/components/workspaces/Modal/ModalAlert/ModalAlert.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { FC } from 'react';
+import { Modal, Alert } from 'antd';
+
+export interface IModalAlertProps {
+  headTitle: string;
+  showModal: boolean;
+  alertMessage: string;
+  alertDescription: string;
+  alertType: 'info' | 'success' | 'error' | 'warning' | undefined;
```

Comment:
I tried the `undefined` value and the app crashes. I don't know if it necessary, if it is you need to handle it properly

Reply:
I removed it